task/sast-snyk-check/0.3/MIGRATION.md

```diff
@@ -0,0 +1,15 @@
+# Migration from 0.2 to 0.3
+
+Version 0.3:
+
+- The `--severity-threshold` argument has been introduced and enabled by default with "high" value. Only high or superior vulnerabilities will be shown. Possible values for this argument are: `low`, `medium` and `high`.
+- The results are parsed using `csgrep` and they are uploaded as results with a generated fingerprint.
+- There are no default arguments as "--all-projects --exclude=test*,vendor,deps" are ignored by Snyk Code
+- The results are uploaded as SARIF files to the registry
+- SARIF produced by Snyk Code is not included in the CI log.
+- The findings are filtered using the known-false-positives repository (https://gitlab.cee.redhat.com/osh/known-false-positives/)
+- The KFP_GIT_URL environment variable has been introduced to indicate the repository to filter false positives. If this variable is left empty, the results won't be filtered.
```

Comment:
I think we should refer to `KFP_GIT_URL` as a parameter here.

Reply:
Fixed